docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -12,10 +12,16 @@ Instead, Teleport Cloud users should consult the following guide, which shows yo
 
 Teleport can provide secure, unified access to your Kubernetes clusters. This guide will show you how to:
 
+<ScopedBlock scope={["enterprise"]}>
+- Install the Teleport Enterprise License.
```

Comment:
Is this a distinct task from deploying Teleport Enterprise in a Kubernetes cluster?

Reply:
Basically installing the license as a secret.  We can remove it if you find it confusing.